### PR TITLE
docs: fixed tutorial data table component

### PR DESCRIPTION
**Pull request by william-monroy**
### Data Table component

Captura de pantalla 2023-10-02 a la(s) 12 55 46 a m

Missing highlight on line 29 to close `state` prop.

### Contribution guidelines

Update steps to easily contribute to this project.

### CODE REVIEW

**Review by SoyDiego — APPROVED** [review]

**Review by jocarrd — APPROVED** [review]

**Comment by william-monroy** [comment]
Hi @elliot, could you review my pull-request please? Thanks :octocat:

File: apps/www/content/docs/components/data-table.mdx [hunk 1]
```diff
@@ -463,7 +463,7 @@ Let's make the email column sortable.
 
 ### Update `<DataTable>`
 
-```tsx showLineNumbers title="app/payments/data-table.tsx" showLineNumbers {3,6,10,18,25-28}
```
Comment (shadcn) [inline apps/www/content/docs/components/data-table.mdx:466]: This looks good. Can we do small PRs please? Would be great if you could break the other changes in different PRs. You can keep this one. We'll merge. Thank you.

Comment (william-monroy) [reply apps/www/content/docs/components/data-table.mdx:466]: It is an honor to address you, thank you very much for reviewing my PR. I will roll back the changes from the `CONTRIBUTING.md` file to the latest version of `main` and make a different PR for it keeping only the `data-table.mdx` correction for this PR.

**Comment by william-monroy** [comment]
I made the changes, please could you review them? Thank you very much.

**Comment by shadcn** [comment]
Will merge https://github.com/shadcn-ui/ui/pull/1830. Closing this one. Thanks @william-monroy
